src/test/java/org/wso2/carbon/module/swiftiso20022/MT940FormatValidatorTests.java

```diff
@@ -94,66 +94,67 @@ public void testMT940FormatValidatorForInvalidPayload() throws Exception {
     @Test(dataProvider = "invalidReferenceDataProvider",
             dataProviderClass = MT940FormatValidatorTestConstants.class)
     public void testValidateTransactionReference(String reference) {
-        ErrorModel error = MT940ValidationUtils.validateReference(reference,
-                ConnectorConstants.TRANSACTION_REF);
+        ValidationResult error = MT940ValidationUtils.validateReference(reference,
```

Comment:
```suggestion
        ValidationResult validationResult = MT940ValidationUtils.validateReference(reference,
```

Reply:
Fixed in https://github.com/wso2-extensions/mediation-swift-iso20022-module/pull/6/commits/7f2058c938da71ec80755d296fd2d3d67541e678